app/ts/background/simulationUpdating.ts

```diff
@@ -135,7 +135,7 @@ export const simulateGovernanceContractExecution = async (pendingTransaction: Pe
 			simulatedBlocks: [{
 				signedMessages: [],
 				stateOverrides: {},
-				blockTimestamp: new Date(parentBlock.timestamp.toTimeString() + 12 * 1000),
+				blockTimestamp: bigintSecondsToDate((dateToBigintSeconds(parentBlock.timestamp) + getBlockTimeManipulationSeconds(DEFAULT_BLOCK_MANIPULATION.deltaToAdd, DEFAULT_BLOCK_MANIPULATION.deltaUnit))),
```

Comment:
I wonder if we should just store dates as seconds everywhere and only turn them into `Date` instances at the edges?  It seems silly to have code like `bigintSecondsToDate((dateToBigintSeconds(...`

Reply:
yeah, I added issue for this, it's better